public/css/doctors.css

```diff
@@ -0,0 +1,100 @@
+*{
```

Comment:
this * one is the same in all of your css files so make a default file for this and import it in your head.hbs